prestoadmin/server.py

```diff
@@ -104,15 +314,31 @@ def install(local_path):
     jvm.config
 
     Parameters:
-        local_path -            Absolute path to the presto rpm to be installed
-
-        --nodeps -              (optional) Flag to indicate if server install
-                                should ignore checking Presto rpm package
-                                dependencies. Equivalent to adding --nodeps
-                                flag to rpm -i.
+        rpm_specifier - String specifying location of presto rpm to copy and install
+                        to nodes in the cluster. The string can specify a presto rpm
+                        in the following ways:
+
+                        1.  Path to a local copy
+                        2.  Url to download
+                        3.  Version number to download
+                        4.  'latest' to download the most latest release
+
+                        Before downloading an rpm, install will attempt to find a local
+                        copy with a matching version number to the requested rpm. If such
+                        a match is found, it will use the local copy instead of downloading
+                        the rpm again.
+
+        --rpm-source -  String specifying where the presto rpm should be downloaded from if
+                        necessary. By default, this is set to 'facebook'.
+
+        --nodeps -      (optional) Flag to indicate if server install
```

Comment:
Larger question than this PR: Should we expose this in the docstrings or leave it hidden? What's the use-case for this?